selfdrive/car/honda/values.py

```diff
@@ -75,6 +75,7 @@ class CAR:
   CIVIC = "HONDA CIVIC 2016"
   CIVIC_BOSCH = "HONDA CIVIC (BOSCH) 2019"
   CIVIC_BOSCH_DIESEL = "HONDA CIVIC SEDAN 1.6 DIESEL 2019"
+  CIVIC_2022 = "HONDA CIVIC 2022"
```

Comment:
we should probably start moving to using generations like the VW port

Reply:
I'm fine with making a migration later on if it turns out all new Hondas (/Civics) are like this.